service/support/filter-parser.js

```diff
@@ -38,11 +38,15 @@ const parseInternal = filter => {
       return `${filter.fieldName} >= ${mysql.escape(mapValue(filter.value))}`
     case '$hasSome':
     case '$contains': {
-      const list = filter.value
-        .map(mapValue)
-        .map(date => mysql.escape(date, null, null))
-        .join(', ')
-      return list ? `${filter.fieldName} IN (${list})` : EMPTY
+      if (typeof filter.value === 'string') {
```

Comment:
Don't see a test for this branch (with $contains filter and string as filter.value), maybe it's worth adding it?